conda-forge.yml

```diff
@@ -6,4 +6,6 @@ appveyor:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 provider:
   win: azure
+  linux_ppc64le: azure
+  linux_aarch64: azure
```

Comment:
Can you use `default` instead of `azure`?

Reply:
setting to default triggers a build to travis-ci.org, which errors out without initiating a build. Hence had modified to use azure. Changed back to "default" as per suggestion

Reply:
@isuruf, I've updated the value to default as suggested